src/main/java/xades4j/utils/TimeStampDigestInputImpl.java

```diff
@@ -90,13 +91,9 @@ private void addToDigestInput(XMLSignatureInput refData, Document doc) throws Ca
                 // Fall through to add the bytes resulting from the canonicalization.
             }
 
-            if (refData.isByteArray())
-            {
-                digestInput.write(refData.getBytes());
-            } else if (refData.isOctetStream())
-            {
-                StreamUtils.readWrite(refData.getOctetStream(), digestInput);
-            }
+            digestInput.write(refData.getBytes());
+            digestInput.close();
+

```

Comment:
this lines need to be watched carefully, as I don't know for sure, if that's ok

Reply:
Ok, i'll revert this bump.

Reply:
https://github.com/luisgoncalves/xades4j/issues/277